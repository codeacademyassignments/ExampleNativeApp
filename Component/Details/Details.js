import { View, Text } from 'react-native';
import React from 'react';
import DetailsStyle from './Details.style';

class Details extends React.Component {
  render() {
    return (
      <View style={DetailsStyle.greetingsView}>
        <Text style={DetailsStyle.greetingsText}>Welcome to React Native !!</Text>
      </View>
    );
  }
}

export default Details;
