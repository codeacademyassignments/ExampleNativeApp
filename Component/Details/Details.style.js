import { StyleSheet } from 'react-native';

export default StyleSheet.create({
  greetingsView: {
    flex: 2,
    alignItems: 'center',
    justifyContent: 'space-around',
  },
  greetingsText: {
    color: 'blue',
  },
});
