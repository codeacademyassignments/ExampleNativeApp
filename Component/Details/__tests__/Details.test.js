import renderer from 'react-test-renderer';
import React from 'react';
import Details from '../Details';

describe('Details', () => {
  it('should match with old snapshot', () => {
    const render = renderer.create(<Details />).toJSON();
    expect(render).toMatchSnapshot();
  });
});
