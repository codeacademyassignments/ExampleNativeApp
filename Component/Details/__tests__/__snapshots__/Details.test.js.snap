// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Details should match with old snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 2,
      "justifyContent": "space-around",
    }
  }
>
  <Text
    style={
      Object {
        "color": "blue",
      }
    }
  >
    Welcome to React Native !!
  </Text>
</View>
`;
